app/screens/home/channel_list/categories_list/header/index.ts

```diff
@@ -23,6 +23,14 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
 
     const enableOpenServer = observeConfigBooleanValue(database, 'EnableOpenServer');
 
+    const enableGuestAccounts = observeConfigBooleanValue(database, 'EnableGuestAccounts');
+
+    const buildEnterpriseReady = observeConfigBooleanValue(database, 'BuildEnterpriseReady');
```

Comment:
I have never used nor seen this property `BuildEnterpriseReady`, can you explain how it works?